package.json

```diff
@@ -2,7 +2,7 @@
   "name": "openapi-workspaces",
   "license": "MIT",
   "private": true,
-  "version": "0.47.13",
+  "version": "0.47.14",
```

Comment:
going to publish again because our older version seems to have a json pointer helpers version already published https://github.com/opticdev/optic/actions/runs/5591403588